homeassistant/components/androidtv/media_player.py

```diff
@@ -280,13 +272,17 @@ def service_adb_command(service):
                     output,
                 )
 
-    hass.services.register(
+    hass.services.async_register(
         ANDROIDTV_DOMAIN,
         SERVICE_ADB_COMMAND,
         service_adb_command,
         schema=SERVICE_ADB_COMMAND_SCHEMA,
     )
 
+    platform.async_register_entity_service(
+        SERVICE_LEARN_SENDEVENT, SERVICE_LEARN_SENDEVENT_SCHEMA, "learn_sendevent",
```

Comment:
Passing an empty dict as schema will generate an entity id schema automatically that also supports area ids.

Reply:
I can address this later today.